src/components/nav-icon-item/nav-icon-item.tsx

```diff
@@ -0,0 +1,63 @@
+import { AnchorHTMLAttributes, ButtonHTMLAttributes, FC, MouseEventHandler, ReactNode } from 'react'
+import { ElAnchorNavIconItem, ElButtonNavIconItem } from './styles'
+
+interface CommonNavIconItemProps {
```

Comment:
**issue:** I think `aria-label` should be explicitly defined here as a required prop.

Reply:
> think aria-label should be explicitly defined here as a required prop

yeah, I think this could be good for future NavIconItem stuff, especially for the tooltip part beside of a11y